CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # GraphDB Azure Terraform Module Changelog
 
+## 1.3.4
```

Comment:
These updates should not be here if you rebased properly, also you need to update the Changelog with your changes

Reply:
This is still valid.
Also you need to add a new version with your changes to the CHANGELOG.md